prac_05/hex_colours.py

```diff
@@ -0,0 +1,9 @@
+HEX_COLOURS = {"AliceBlue": "#f0f8ff", "aquamarine1": "#7fffd4"}
+
+colour = input("Enter name of colour: ")
+while colour != "":
+    if colour in HEX_COLOURS:
+        print(HEX_COLOURS[colour])
+    else:
+        print("Invalid colour name!")
+    break
```

Comment:
Do you really need to break out of this loop or could you simply re-ask the user for another colour?